jobs/scripts/glusto/setup-glusto.yml

```diff
@@ -309,6 +309,12 @@
   - name: Upgrading pip
     command: "pip3 install --upgrade pip"
 
+  - name: Install pytest
+    command: "pip3 install pytest==6.1.2"
```

Comment:
Shouldn't it be `>=` ?

Reply:
The issue is with pytest 6.2.1 which is the latest version of pytest which was released day before yesterday, hence till this issue is resolved in the upcoming releases of pytest, we have temporary added code to install only pytest 6.1.2. Hence it's intentionally kept `==` and not `>=`.